src/FSharpSource.targets

```diff
@@ -108,6 +108,11 @@
     <IntermediateOutputPath>obj\$(Configuration)\$(TargetFramework)\</IntermediateOutputPath>
   </PropertyGroup>
 
+  <!-- Produce portable pdbs -->
+  <PropertyGroup Condition="'$(Configuration)'!='Proto'">
+    <OtherFlags>$(OtherFlags) --debug:portable</OtherFlags>
```

Comment:
It is not possible. PE file can only reference a single PDB. Once implemented our symbol server publishing tool will however be able to publish both Windows and Portable PDB into the symbol store so that consumer (debuggers) can fetch the most appropriate format for their platform. To publish both PDBs you'd compile your project with Windows PDB (on Windows machine). The symbol publishing tool will have an option to translate Windows PDB to Portable PDB and upload both.